web/src/ErrorPane.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { PureComponent } from "react"
+import AnsiLine from "./AnsiLine"
+import "./ErrorPane.scss"
+import { zeroTime } from "./time"
+import { Build } from "./types"
+
+class ErrorResource {
+  public name: string
+  public buildHistory: Array<Build>
+  public resourceInfo: ResourceInfo
+
+  constructor(resource: any) {
+    this.name = resource.Name
+    this.buildHistory = resource.BuildHistory
+    if (resource.ResourceInfo) {
+      this.resourceInfo = {
+        podCreationTime: resource.ResourceInfo.PodCreationTime,
+        podStatus: resource.ResourceInfo.PodStatus,
+        podRestarts: resource.ResourceInfo.PodRestarts,
+        podLog: resource.ResourceInfo.PodLog,
+      }
+    } else {
+      this.resourceInfo = {
+        podCreationTime: zeroTime,
+        podStatus: "",
+        podRestarts: 0,
+        podLog: "",
+      }
+    }
+  }
+}
+
+type ResourceInfo = {
+  podCreationTime: string
+  podStatus: string
+  podRestarts: number
+  podLog: string
+}
+
+type ErrorsProps = {
+  resources: Array<ErrorResource>
+}
+
+class ErrorPane extends PureComponent<ErrorsProps> {
+  render() {
+    let el: JSX.Element = <p>No errors</p>
+    let errorElements: Array<JSX.Element> = []
+    this.props.resources.forEach(r => {
+      if (
+        r.resourceInfo.podStatus === "Error" ||
+        r.resourceInfo.podStatus === "CrashLoopBackOff"
+      ) {
+        errorElements.push(
+          <li key={"resourceInfoError" + r.name}>{r.resourceInfo.podLog}</li>
+        )
+      } else if (r.resourceInfo.podRestarts > 0) {
+        errorElements.push(
+          <li key={"resourceInfoPodCrash" + r.name}>
+            <p>{`${r.name} has container restarts: ${
+              r.resourceInfo.podRestarts
+            }.`}</p>
+            <p>{`Last log line: ${r.resourceInfo.podLog}`}</p>
+          </li>
+        )
+      }
+      r.buildHistory.forEach((b, i) => {
```

Comment:
For now, we should only show at most one build error for each resource. 

In the future, we can explore showing all the accumulated errors — but this will require more explanation in the UI, and possibly a "clear" button.